tests/tests_ssh_reboot.yml

```diff
@@ -151,3 +146,10 @@
           - __ssh_dir_stat_before.stat.mode == __ssh_dir_stat_after.stat.mode
           - __ssh_dir_stat_before.stat.uid == __ssh_dir_stat_after.stat.uid
           - __ssh_dir_stat_before.stat.gid == __ssh_dir_stat_after.stat.gid
+
+    - name: Delete user
+      user:
+        name: "{{ kdump_ssh_user }}"
+        state: absent
+      register: __user_delete
+      until: __user_delete is success
```

Comment:
The default value for “retries” is 3 and “delay” is 5, is it enough here?
https://docs.ansible.com/ansible/latest/playbook_guide/playbooks_loops.html#retrying-a-task-until-a-condition-is-met

Reply:
I think so - the error is very difficult to reproduce in CI